packages/fiori/src/WizardTab.ts

```diff
@@ -0,0 +1,212 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { isSpace, isEnter, isSpaceShift } from "@ui5/webcomponents-base/dist/Keys.js";
+import type { ITabbable } from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import Icon from "@ui5/webcomponents/dist/Icon.js";
+import WizardTabTemplate from "./generated/templates/WizardTabTemplate.lit.js";
+import WizardTabCss from "./generated/themes/WizardTab.css.js";
+
+type WizardTabInfo = {
+	[key: string]: string,
+}
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ * Private component, used internally by the <code>ui5-wizard</code>
+ * to represent a "step" in the navigation header of the <code>ui5-wizard</code>.
+ *
+ * <h3>Usage</h3>
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/WizardTab.js";</code> (imported with <ui5-wizard>)
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.fiori.WizardTab
+ * @extends sap.ui.webc.base.UI5Element
+ * @tagname ui5-wizard-tab
+ * @private
+ */
+
+@customElement("ui5-wizard-tab")
+
+/**
+ * Fired when clicking on none disabled step.
+ *
+ * @event sap.ui.webc.fiori.WizardTab#selection-change-requested
+ * @private
+ */
+@event("selection-change-requested")
+
+class WizardTab extends UI5Element implements ITabbable {
+	/**
+	 * Defines the <code>icon</code> of the step.
+	 * @type {string}
+	 * @name sap.ui.webc.fiori.WizardTab.prototype.icon
```

Comment:
I am not sure about that, but I think that the guidance was to not set @name when the property is private. Excuse me, if I am wrong but could you please check? The comment is valid for all the properties of WizardTab

Reply:
The reason to not add @name to private properties because JSDoc plugin will exclude them even if they have it. It's a problem when public properties doesn't have @name because they won't appear into the documentation.